src/org/labkey/test/components/SetDefaultPageClass.java

```diff
@@ -0,0 +1,19 @@
+package org.labkey.test.components;
+
+import org.labkey.test.Locator;
+import org.labkey.test.pages.LabKeyPage;
+import org.openqa.selenium.WebDriver;
+
+public class SetDefaultPageClass extends LabKeyPage
```

Comment:
I believe this "Set Default" page only applies within the study module for datasets. Should this move down into the `test/components/study` dir?

Reply:
Done